build-and-deploy-cloud-functions.sh

```diff
@@ -19,16 +19,16 @@ gcloud --project=${PROJECT_ID} functions deploy \
     --vpc-connector=projects/${PROJECT_ID}/locations/us-central1/connectors/appengine-default-connect
 
 
-echo "Deploying kit boston tracking to ${PROJECT_ID}"
-gcloud --project=${PROJECT_ID} functions deploy \
-    tbos-kit-ups-tracker \
-    --entry-point=org.broadinstitute.dsm.jobs.TestBostonUPSTrackingJob \
-    --runtime=java11 \
-    --trigger-topic=cf-kit-tracking \
-    --timeout=540\
-    --source=target/deployment \
-    --set-env-vars="PROJECT_ID=${PROJECT_ID},SECRET_ID=cf-kit-tracker,STUDY_MANAGER_SCHEMA=${STUDY_MANAGER_SCHEMA},STUDY_SERVER_SCHEMA=${STUDY_SERVER_SCHEMA}" \
-    --vpc-connector=projects/${PROJECT_ID}/locations/us-central1/connectors/appengine-default-connect
+#echo "Deploying kit boston tracking to ${PROJECT_ID}"
+#gcloud --project=${PROJECT_ID} functions deploy \
+#    tbos-kit-ups-tracker \
+#    --entry-point=org.broadinstitute.dsm.jobs.TestBostonUPSTrackingJob \
+#    --runtime=java11 \
+#    --trigger-topic=cf-kit-tracking \
+#    --timeout=540\
+#    --source=target/deployment \
+#    --set-env-vars="PROJECT_ID=${PROJECT_ID},SECRET_ID=cf-kit-tracker,STUDY_MANAGER_SCHEMA=${STUDY_MANAGER_SCHEMA},STUDY_SERVER_SCHEMA=${STUDY_SERVER_SCHEMA}" \
+#    --vpc-connector=projects/${PROJECT_ID}/locations/us-central1/connectors/appengine-default-connect
 
```

Comment:
This part deploys the `TestBostonUPSTrackingJob` which we haven't changed, I commented this part out so when we deploy only the changed CF will be redeployed